File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Client/src/models/ReIdentifications/IdentificationItem.ts
export class IdentificationItem {
    public rowNumber: string;
    public identifier: string;
    public message: string;
    public hasAccess?: boolean;
    public isReidentified?: boolean;

    constructor(identificationItem: IdentificationItem) {
        this.rowNumber = identificationItem.rowNumber ? identificationItem.rowNumber : "";
        this.identifier = identificationItem.identifier || "";
        this.message = identificationItem.message || "";
        this.hasAccess = identificationItem.hasAccess || false;
        this.isReidentified = identificationItem.isReidentified || false;
    }
}

File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Server.Tests.Integration/Models/ReIdentifications/IdentificationItem.cs
﻿// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

namespace ISL.ReIdentification.Portals.Server.Tests.Integration.ReIdentification.Models.ReIdentifications
{
    public class IdentificationItem
    {
        public string RowNumber { get; set; } = string.Empty;
        public string Identifier { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
        public bool HasAccess { get; set; } = false;
        public bool IsReidentified { get; set; } = false;
    }
}
